stable/datadog/README.md

```diff
@@ -116,3 +118,11 @@ datadog:
           port: 6379
 ```
 
+### leader election
+
+The Datadog Agent supports built in leader election option for the Kubernetes event collector As of 5.17.
+This feature relies on ConfigMaps, enabling this flag will grant Datadog Agent get, list, delete and create access to the ConfigMap resource.
+See the full [RBAC](https://github.com/DataDog/integrations-core/tree/master/kubernetes#gathering-kubernetes-events)
+Agents coordinate by performing a leader election among members of the Datadog DaemonSet through kubernetes to ensure only one leader agent instance is gathering events at a given time.
+*This functionality is disabled by default.*
+The `datadog.leaderLeaseElection` is the duration for which a leader stays elected. It should be > 30 seconds. The longer it is, the less hard your agent hits the apiserver with requests, but it also means that if the leader dies (and under certain conditions) there can be an event blackout until the lease expires and a new leader takes over.
```

Comment:
`the less hard your agent hits` --> `the less frequently your agents hit` (I know I wrote this sentence, sorry 😅)

`there can be an event blackout` --> `events can be missed`